Upgrade validators - systemd (#245)
moondance-labs/tanssi-docs: node-operators/validators/operational-tasks/upgrade-your-node/upgrade-systemd.md

---
title: Upgrade Your Validator Systemd Node
description: Follow these instructions to update your Tanssi validator node running via Systemd to the latest version of the Tanssi client software.
icon: simple-linux
---

# Upgrade Your Node Running via Systemd

## Introduction {: #introduction }

Keeping your node up-to-date is an important part of being a Tanssi validator. Not only does it help ensure that your node stays healthy, but it also contributes to keeping the entire Tanssi Network running smoothly.

This tutorial covers upgrading your Tanssi validator node that was configured using Systemd. It assumes you have already set up your account and launched a [validator node using Systemd](/node-operators/validators/onboarding/run-a-validator/validators-systemd/){target=\_blank}.

--8<-- 'text/node-operators/github-release-notifications.md'

## Upgrading Your Systemd Node {: #upgrading-your-systemd-node }

> If you're running your validator via the Systemd service, you'll need to take a few steps to upgrade your node properly. In short, you'll need to stop the service, replace the Tanssi binary with the updated version, and then start the service.

Reviewer: ```suggestion
If you're running your validator via the Systemd service, you must take steps to upgrade your node correctly. In short, you'll need to stop the service, replace the Tanssi binary with the updated version, and restart the service.
```

---
title: Upgrade Your Validator Systemd Node
description: Follow these instructions to update your Tanssi validator node running via Systemd to the latest version of the Tanssi client software.
icon: simple-linux
---

# Upgrade Your Node Running via Systemd

## Introduction {: #introduction }

> Keeping your node up-to-date is an important part of being a Tanssi validator. Not only does it help ensure that your node stays healthy, but it also contributes to keeping the entire Tanssi Network running smoothly. Tanssi validator operators can subscribe to [GitHub notifications](#subscribe) to be alerted about new client release versions.

Reviewer 2: The last sentence doesn't seem right because you have a reminder in the index, and then two more reminders on this content page. It might be better to remove it and include a single admonition here instead.

    chmod +x ./tanssi-relay*
    ```

=== "AMD Zen3"

    ```bash
    wget https://github.com/moondance-labs/tanssi/releases/download/{{ networks.dancebox.client_version }}/tanssi-relay-znver3 -O tanssi-node && \
    wget https://github.com/moondance-labs/tanssi/releases/download/{{ networks.dancebox.client_version }}/tanssi-relay-execute-worker-znver3 -O tanssi-node && \
    wget https://github.com/moondance-labs/tanssi/releases/download/{{ networks.dancebox.client_version }}/tanssi-relay-prepare-worker-znver3 -O tanssi-node && \
    chmod +x ./tanssi-relay*
    ```

You can restart your Tanssi Systemd service with the following command:

```bash
systemctl start tanssi.service
```

The node will resume syncing blocks from where it left off when the Systemd service was stopped. To verify that it is running correctly, you can use the following command to check the logs:

```bash
systemctl status tanssi.service
```

And that's it! You've successfully upgraded your Tanssi node.

## Subscribe to Release Notifications {: #subscribe }

> You can subscribe to email notifications of new releases by clicking **Watch** on the [Tanssi GitHub repo](https://github.com/moondance-labs/tanssi){target=\_blank} and selecting **Custom** notifications and checking the box for **Releases**.

Reviewer 2: I believe this will work better as an admonition, wdyt?

> You can subscribe to email notifications of new releases by clicking **Watch** on the [Tanssi GitHub repo](https://github.com/moondance-labs/tanssi){target=\_blank} and selecting **Custom** notifications and checking the box for **Releases**.

Reviewer 2: I believe this will work better as an admonition, wdyt?

Author: Unified the admonition amongst sequencers and validators, and their index pages
